hack/migrate/main.go

```diff
@@ -13,6 +13,7 @@ var migrate = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		if err := duty.Migrate(api.Config{
 			ConnectionString: connection,
+			Postgrest:        api.DefaultConfig.Postgrest,
```

Comment:
resolves: `ERR+1 failed to grant postgrest roles: ERROR: syntax error at or near "TO" (SQLSTATE 42601)`